src/components/FileUploader/FileUploader-story.js

```diff
@@ -10,7 +10,7 @@ const tooltipProps = {
   text: 'only .jpg files at 500mb or less',
   iconName: 'info',
   menuOffset: {
-    top: -128,
+    top: 8,
```

Comment:
these offsets probably shouldn't need to be specified, I believe there is an issue with how the floating menu is measuring the trigger element